content/en/docs/tutorials/security/ns-level-pss.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Apply Pod Security Standards at the Namespace Level
+content_type: tutorial
+weight: 10
+---
+
+{{% alert title="Note" %}}
+This tutorial applies only for new clusters.
+{{% /alert %}}
+
+Pod Security admission (PSA) is enabled by default in v1.23 and later, as it [graduated
+to beta](/blog/2021/12/09/pod-security-admission-beta/). Pod Security Admission
+is an admission controller that applies 
+[Pod Security Standards](docs/concepts/security/pod-security-standards/) 
+when pods are created. In this tutorial, you will enforce the `baseline` Pod Security Standard,
+one namespace at a time.
+
+You can also apply Pod Security Standards to multiple namespaces at once at the cluster
+level. For instructions, refer to [Apply Pod Security Standards at the cluster level](/docs/tutorials/security/cluster-level-pss).
+## {{% heading "prerequisites" %}}
+
+Install the following on your workstation:
+
+- [KinD](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+## Create cluster
+
+1. Create a `KinD` cluster as follows:
+
+    ```shell
+    kind create cluster --name psa-ns-level --image kindest/node:v1.23.0
+    ```
+   The output is similar to this:
+    ```
+    Creating cluster "psa-ns-level" ...
+     ✓ Ensuring node image (kindest/node:v1.23.0) 🖼 
+     ✓ Preparing nodes 📦  
+     ✓ Writing configuration 📜 
+     ✓ Starting control-plane 🕹️ 
+     ✓ Installing CNI 🔌 
+     ✓ Installing StorageClass 💾 
+    Set kubectl context to "kind-psa-ns-level"
+    You can now use your cluster with:
+    
+    kubectl cluster-info --context kind-psa-ns-level
+    
+    Not sure what to do next? 😅  Check out https://kind.sigs.k8s.io/docs/user/quick-start/
+    ```
+
+1. Set the kubectl context to the new cluster:
+    ```shell
+    kubectl cluster-info --context kind-psa-ns-level
+    ```
+   The output is similar to this:
+    ```
+    Kubernetes control plane is running at https://127.0.0.1:50996
+    CoreDNS is running at https://127.0.0.1:50996/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+    
+    To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+    ```
+
+## Create a namespace
+
+Create a new namespace called `example`:
+
+```shell
+kubectl create ns example
+```
+The output is similar to this:
+```
+namespace/example created
+```
+
+## Apply Pod Security Standards
+
+1. Enable Pod Security Standards on this namespace using labels supported by
+   built-in Pod Security Admission. In this step we will warn on baseline pod
+   security standard as per the latest version (default value)
+
+    ```shell
+    kubectl label --overwrite ns example \
+      pod-security.kubernetes.io/warn=baseline \
+      pod-security.kubernetes.io/warn-version=latest
+    ```
+
+2. Multiple pod security standards can be enabled on any namespace, using labels.
```

Comment:
it might be worth noting that if this is an existing namespace that already contains workloads, using --dry-run=server is recommended first to determine if the new policy levels will disrupt existing workloads